### PR TITLE
Add configuration information for paypload_reset

### PR DESCRIPTION
**Pull request by ShadowJonathan**
## Proposed change

This adds the necessary configuration documentation for MQTT Device Tracker auto-zoning.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/83952

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jbouwh** [comment]
Please also update the abbreviations list for Mqtt discovery, found at the integration page.

File: source/_integrations/device_tracker.mqtt.markdown [hunk 1]
```diff
@@ -74,6 +74,11 @@ payload_not_home:
   required: false
   type: string
   default: "not_home"
+payload_auto_zone:
+  description: The payload value that will have the device's location automatically derived from home assistant's zones.
```
Comment (frenck) [inline source/_integrations/device_tracker.mqtt.markdown]:
```suggestion
  description: The payload value that will have the device's location automatically derived from Home Assistant's zones.
```

File: source/_integrations/device_tracker.mqtt.markdown [hunk 2]
```diff
@@ -205,6 +210,11 @@ payload_not_home:
   required: false
   type: string
   default: not_home
+payload_auto_zone:
+  description: The payload value that will have the device's location automatically derived from home assistant's zones.
```
Comment (frenck) [inline source/_integrations/device_tracker.mqtt.markdown]:
```suggestion
  description: The payload value that will have the device's location automatically derived from Home Assistant's zones.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by ShadowJonathan** [comment]
Update: I rebased on `next` and changed `auto_zone` to `reset` as per [this review](https://github.com/home-assistant/core/pull/83952#discussion_r1051980416)

**Review by jbouwh — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**